### PR TITLE
Chore/named imports

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] Add the Clubhouse Story ID: [ch23766]

I also have a branch migrating all underscore references to named imports. But this PR is big enough without those, so we'll start with this.

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -87,3 +88,15 @@ document.addEventListener('DOMContentLoaded', () => {
 
   getConfig();
 });
+
+// For use of escaping a string for within a regex
```
Comment (LisaManresa) [inline src/js/index.js]: Might not be the right place for these. Any other suggestions @paulfalgout?

File: src/js/index.js [hunk 2]
```diff
@@ -87,3 +88,15 @@ document.addEventListener('DOMContentLoaded', () => {
...
+// http://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex#6969486
+RegExp.escape = function(str) {
+  return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
+};
+
+// Underscore uses {{ }} and {{{ }}}
+_.templateSettings = {
```
Comment (paulfalgout) [inline src/js/index.js]: I think maybe we can remove this entirely?  Are we really using underscore templates somewhere?

File: src/js/index.js (hunk 1 above)
Comment (paulfalgout) [reply src/js/index.js]: Yeah I think `base/setup` would be better

File: src/js/utils/formatting/dasherize.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import trim from 'js/utils/formatting/trim';
```
Comment (paulfalgout) [inline src/js/utils/formatting/dasherize.js]: we don't actually seem to be using this.. Should we keep it around?  Maybe

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/index.js (hunk 2 above)
Comment (LisaManresa) [reply src/js/index.js]: I thought I remembered talking about using underscore templates for the patient fields sidebar work. But looking at `patient-template.js` that doesn't seem to be the case and there are no other instances of _.template. So I'll remove this.

File: src/js/utils/formatting/dasherize.js (hunk 3 above)
Comment (LisaManresa) [reply src/js/utils/formatting/dasherize.js]: Ah right, it's used by `slugify()`, but we removed that as well. I can remove this

**Pull request merged**